docs/DesignDocs/SGX_QuoteEx_Integration.md

```diff
@@ -0,0 +1,121 @@
+# Attestation: OE SDK Integration with Intel® SGX SDK quote-ex Library for Generation of Evidence in New Formats
+
+This design document proposes an extension of the OE SDK implementation for integration with the Intel® SGX SDK quote-ex library, for support of generation of evidence in new SGX formats such as Enhanced Privacy ID (EPID).
```

Comment:
nit: Isn't libsgx-quote-ex part of the SGX PSW not the SGX SDK? Or are the terms used interchangeably?

Reply:
quote-ex is a library package as part of the overall SGX SDK. Internally, this library communicates to the AESM daemon via IP / Unix socket. AESM is part of the SGX PSW.